### DISCUSSION TITLE
Disable/override install

### DISCUSSION THREAD

**Discussion by leingang** [latex3/l3build#208] [Q&A] [answered]
Hi! I'm looking at `l3build` for the main purpose of unpacking `.tex` files from a DocStrip file and building them.  My `build.lua` file looks like:

    unpackfiles  = {"*.dtx"}
    typesetfiles = {"*.*.tex"}

I don't want to “install” the files anywhere locally.  I may want to deploy the PDFs to an online destination, or push out `tex` files to an Overleaf project.  I know that I can make custom targets, but can I overwrite the `install` target?  Or can I disable `install`? to avoid copying these files to my local texmf?

I know this isn't the intended use of `l3build`, but I like the interface and framework so I wonder if it can be finagled so.

**Comment by josephwright** [comment, marked as answer]
You should be able to zap it with `target_list.install = null`, which removes `install` from the standard list.

**Comment by leingang** [comment]
@josephwright good idea.  This also seems to work, and leaves the option to explain.

    function dont_install()
        print("Install disabled! Mwa hah hah")
        return 0
    end

    if options["target"] == "install" then
        dont_install()
        os.exit()
    end
